app/src/main/java/com/example/tasky/agenda/data/AgendaRepositoryImpl.kt

```diff
@@ -74,6 +75,20 @@ class AgendaRepositoryImpl @Inject constructor(
         createTask(task = task.toDomain(), isUpdate = true)
     }
 
+    override suspend fun insertEvent(event: AgendaItem.Event) {
+        event.attendees.map {
+            agendaDao.insertAttendee(it.toEntity())
+            agendaDao.insertEventAttendeeCrossRef(
+                EventAttendeesCrossRef(
+                    id = event.id,
+                    userId = it.userId
+                )
+            )
```

Comment:
You're making all these calls sequentially here, so it'll first insert the first attendee, then the second one, etc.
They don't depend on each other, so you can make use of concurrency:
```
supervisorScope {
    event.attendees.map {
        launch {
            // Insert attendee and cross ref
        }
    }.map { it.join() }
}
```